docker-compose.yaml

```diff
@@ -575,7 +575,7 @@ services:
                 "pg-amqp-bridge"
             ]
     graphql:
-        image: hasura/graphql-engine:v2.8.4
+        image: hasura/graphql-engine:v2.10.0
```

Comment:
please change in all locations we are using this (other compose files plus the k8s manifests)